types/storybook__addon-chapters/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for @storybook/addon-chapters 0.6
+// Project: https://github.com/storybooks/storybook, https://github.com/sm-react/storybook-chapters
+// Definitions by: Ian Ker-Seymer <https://github.com/ianks>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { StoryDecorator, Story } from "@storybook/react";
+
+declare module "@storybook/react" {
+    interface Story {
+        addChapter: (name: string, cb: (chapter: Story) => Story) => Story;
+    }
+}
+
+export const withChapters: StoryDecorator;
```

Comment:
It looks like from [the docs](https://www.npmjs.com/package/@storybook/addon-chapters) that there is no export `withChapters`. Did you mean to export `jumpTo`?
Also, it looks like there's a default export as well.